README.md

```diff
@@ -26,23 +20,21 @@ This is a simple yet powerful React-Redux boilerplate for later use as a startin
 
 ## Getting Started
 
-To get React and Redux up and running in another project, follow these simple steps:
+To get this project up and running, follow these simple steps:
 
 1. Clone the repository into your machine (Or download the .zip file and extract).
 2. Open the project's folder.
-3. Copy and paste all files and folders except app_screenshot.png, app_screenshot-2.png, MIT.md, and README.md into the other project's root folder.
-4. Run `npm install`
-5. Run `npm start`
-
-### Considerations
-
-- Intended only for use as a starting point in other projects. Do not copy and paste these files into more mature projects as this will overwrite your package.json and package-lock.json files.
-- This project also includes Bootstrap for styling.
+3. Run `npm install`
+4. Run `npm start`
 
 ### Setup
 
 Follow the steps described before.
 
+### Run tests
+
+In order to run the predefined tests for this project, run `npm test`
+
 ## Authors
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely.